packages/edit/vite.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue2";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  root: "./src",
+  plugins: [vue()],
+  build: {
+    lib: {
+      entry: "./index.ts",
+      name: "TCE-Edit",
+      fileName: "content-element-edit",
+    },
+    rollupOptions: {
+      external: ["vue"],
```

Comment:
If we externalize vue2 and vue3 we will need to figure out how to handle it within `/runtime`